docs/src/main/asciidoc/security.adoc

```diff
@@ -204,9 +204,16 @@ Some extensions such as `OIDC`, `OAuth2`, `SmallRye JWT` have the inlined `Ident
 For example, `quarkus-oidc` uses its own `IdentityProvider` to convert a token to `SecurityIdentity`. 
 
 If you use `Basic` or `Form` HTTP-based authentication then you have to add an `IdentityProvider` which can convert a username and password to `SecurityIdentity`.
-See xref:security-getting-started.adoc[JPA IdentityProvider], xref:security-jdbc.adoc[JDBC IdentityProvider] and xref:security-ldap.adoc[LDAP IdentityProvider] for more information.
 
-You can also use xref:security-testing.adoc#configuring-user-information[User Properties IdentityProvider] for testing.
+* For more information about `Basic` or `Form` HTTP-based authentication, see: 
+** xref:security-getting-started.adoc[JPA IdentityProvider]
+** xref:security-jdbc.adoc[JDBC IdentityProvider]
+** xref:security-ldap.adoc[LDAP IdentityProvider]
+
+* For a a Basic Auth canfiguration walkthrough (using JPA), see:
+** xref:security-getting-started.adoc[Getting Started With Security] guide.
+
+For testing, use xref:security-testing.adoc#configuring-user-information[User Properties IdentityProvider]
```

Comment:
```suggestion
You can also use a xref:security-testing.adoc#configuring-user-information[User Properties IdentityProvider] to enable Basic Authentication however it is recommended to use this 'IdentityProvider' for testing only because the user names, passwords and roles are set directly in `application.properties`.
```